tests/NuGet.Services.AzureSearch.Tests/SearchService/SearchResponseBuilderFacts.cs

```diff
@@ -357,6 +360,88 @@ public void UsesVersionSpecificDownloadCount()
                 Assert.Equal(4, response.Data[0].DownloadCount);
             }
 
+            [Fact]
+            public void SortDownloadsAscUsingAuxilaryFile()
+            {
+                var mockAuxilaryDataDownloads = new Dictionary<string, long>();
+                var rnd = new Random();
+                for (int i = 0; i < _fiveSearchResults.Count; ++i)
+                {
+                    var newDoc = _fiveSearchResults.Results[i];
+                    newDoc.Document.PackageId += i;
+                    newDoc.Document.TotalDownloadCount += i;
+                    newDoc.Document.Title += i;
```

Comment:
I would consider lines 371-373 and 411-414 to the constructor as well since it's duplicated.

Reply:
If the _fiveSearchResults is used in the future (like _searchResult is used), it may require changing the initialization and move it to the tests. Since this behaviour is specific to these 2 tests, I would think that putting it here is better.

Reply:
> If the _fiveSearchResults is used in the future (like _searchResult is used), it may require changing the initialization and move it to the tests. 

Generally optimizing for what's known today is preferable than what may be in the future.  In any case, I don't feel strongly about this so I'll mark this as resolved.